src/Handler.cpp

```diff
@@ -636,7 +645,7 @@ namespace CML {
 
     APtr<JSONFile> conf = new JSONFile();
     conf->Load(fr);
-    RStr base_dir = File::Dirname(fr.GetFilename());
+    base_dir = File::Dirname(fr.GetFilename());
```

Comment:
Doesn't need to be a member.  Not used elsewhere, and this name does not clarify what it is a base directory of.